docs/updating/release-process.md

```diff
@@ -7,22 +7,34 @@ meta-description: Learn how the CKEditor 5 versions are published.
+Each code release is noted in the [changelog](https://github.com/ckeditor/ckeditor5/blob/stable/CHANGELOG.md) and enumerates all changes, additions, and bug fixes that took place, also highlighting if there are any breaking changes (i.e. changes that make the latest release incompatible with the previous ones code-wise). The code packages are released on the [CKEditor 5 npm site](https://www.npmjs.com/package/ckeditor5), as well as updated in the {@link installation/getting-started/predefined-builds#download-options predefined builds} and {@link installation/getting-started/quick-start-other#creating-custom-builds-with-online-builder online builder} and are ready to download.
 
 It is good to follow npm release messages about new packages being published as well as periodically check the changelog.
 
-## Public release
+### Nightly releases
 
-Soon after the code release, a more user-oriented information is published. In a [release blog post](https://ckeditor.com/blog/?category=releases&tags=CKEditor-5) the latest release version is presented in more details, with examples, screencasts, screenshots and links to the new or updated documentation if it brings important information about the release. The release blog post would also provide additional information and context for the changes if needed and direct the user toward migration guides if these were created for the release.
+The `nightly` releases are published daily and are built upon the current codebase. It means they may contain new features and bugfixes, but at the same time &ndash; mistakes and errors. Because of that, `nightly` versions are not fit for production environments. They should be seen more as an experimental approach or testing grounds.
 
-It is good to follow [CKEditor Ecosystem Blog](https://ckeditor.com/blog/) as it also brings other important articles and news about features, changes and planned development. You can also [sign up to the monthly newsletter](https://ckeditor.com/newsletter/) in order to be notified about the latest releases.
+### Alpha and RC releases
+
+Sometimes, to test specific new features or important in the upcoming stable release, `alpha` versions are released, too. While they would typically be based on a tested code, they are still not `stable` and hence the same caution is advised when trying these out.
+
+The release candidate (`rc`) versions are often the ones that will become `stable` during the upcoming release.
+
+The `rc` and `alpha` versions are named by appending a numbered type denominator to the previous stable version. The `nightly` ones are marked with a daily date.
```

Comment:
> Releases will be created daily, so integration tests using nightly packages might fail. **Please create an issue in the CKEditor 5 issue tracker in such a case. We may have merged a regression, and our tests did not catch it.**

I like :point\_up:  @pomek wrote in the issue. We could try to incorporate it here.

Reply:
Added to the  Nightly section in commit.